heat/core/operations.py

```diff
@@ -544,7 +549,620 @@ def sum(x, axis=None, out=None):
 
     return __reduce_op(x, torch.sum, MPI.SUM, axis, out)
 
-  
+
+def merge_means(mu1, n1, mu2, n2):
+    """
+    Function to merge two means by pairwise update
+
+    Parameters
+    ----------
+    mu1 : 1D ht.tensor or 1D torch.tensor
+          Calculated mean
+    n1 : 1D ht.tensor or 1D torch.tensor
+         number of elements used to calculate mu1
+    mu2 : 1D ht.tensor or 1D torch.tensor
+          Calculated mean
+    n2 : 1D ht.tensor or 1D torch.tensor
+         number of elements used to calculate mu2
+
+    Returns
+    -------
+    mean of combined set
+    number of elements in the combined set
+    """
+    if mu1.item() == np.nan:
```

Comment:
does comparison against np.nan work for GPUs?

Reply:
removed the comparison. was only for handling edge cases. now handled elsewhere